pandas/core/series.py

```diff
@@ -3459,7 +3458,7 @@ def sort_values(
         self,
         *,
         axis: Axis = ...,
-        ascending: bool | int | Sequence[bool] | Sequence[int] = ...,
```

Comment:
This is undocumented (and incompatible with the declarations in NDFrame) - but it might work at runtime.